<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modules</class>
 <widget class="QDialog" name="modules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_buble">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pers">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>191</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>651</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10% 30% 50% 70%;
  background: #BC4A3C;
  padding: 20px;
</string>
      </property>
      <property name="text">
       <string>Gestion Fournisseurs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10% 30% 50% 70%;
  background: #BC4A3C;
  padding: 20px;
</string>
      </property>
      <property name="text">
       <string>Gestion chantiers</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10% 30% 50% 70%;
  background: #BC4A3C;
  padding: 20px;
</string>
      </property>
      <property name="text">
       <string>Gestion Matériels</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_10">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10% 30% 50% 70%;
  background: #BC4A3C;
  padding: 20px;
</string>
      </property>
      <property name="text">
       <string>Gestion Employées</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_11">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10% 30% 50% 70%;
  background: #BC4A3C;
  padding: 20px;
</string>
      </property>
      <property name="text">
       <string>Gestion Matières premières</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_12">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10% 30% 50% 70%;
 background: #BC4A3C;
 padding: 20px;
</string>
      </property>
      <property name="text">
       <string>Gestion Patenaires</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_backi">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>841</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_backi</zorder>
  <zorder>label_buble</zorder>
  <zorder>label_pers</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
